appinfo/info.xml

```diff
@@ -27,7 +27,7 @@
 	<screenshot>https://raw.githubusercontent.com/nextcloud/user_retention/main/docs/screenshot.png</screenshot>
 
 	<dependencies>
-		<nextcloud min-version="25" max-version="27" />
+		<nextcloud min-version="25" max-version="28" />
```

Comment:
let's branch off the previous version, so the UI look is aligned with the rest of the server?
```suggestion
		<nextcloud min-version="28" max-version="28" />
```
Or at least drop the EOL version:
```suggestion
		<nextcloud min-version="26" max-version="28" />
```

Reply:
I don't think it's worth doing this. The success/error dialogs didn't change with `@nextcloud/dialogs` 5. Also I just tested this branch on `stable27` and it works flawlessly. I agree with dropping 25 support though :+1: